packages/kamino-sdk/src/utils/types.ts

```diff
@@ -170,3 +170,12 @@ export interface VaultParameters {
   feeTier: Decimal;
   rebalancingParameters: RebalanceFieldInfo[];
 }
+
+export interface LiquidityDistribution {
+  data: LiquidityForPrice[];
```

Comment:
I think `data` naming is a bit too generic, I would consider renaming this to liquidities, distributions, liquidityDistributions or something similar. Easier to understand when reading the code that uses this.